conf/docker-aio/readme.txt

```diff
@@ -0,0 +1,23 @@
+first pass docker all-in-one image, intended for running integration tests against.
+
+Could be potentially usable for normal development as well.
+
+
+Initial setup (aka - do once):
+- Do surgery on glassfish4 and solr4.6.0 following guides, place results in `conf/docker-aio/dv/deps` as `glassfish4dv.tgz` and `solr-4.6.0dv.tgz` respectively.
+
+Per-build:
+- `cd conf/docker-aio`, and run `0prep.sh` to copy files for integration test data into docker build context; `0prep.sh` will also build the war file and installation zip file
+- build the docker image: `docker build -t dv0 -f c7.dockerfile .`
+
+- Run image: `docker run -d -p 8083:8080 --name dv dv0` (aka - forward port 8083 locally to 8080 in the container)
+- Installation (integration test): `docker exec -it dv /opt/dv/setupIT.bash`
+- Installation (non-interactive, uses `conf/docker-aio/default.config`): `docker exec -it dv /opt/dv/install.bash`
```

Comment:
I'm confused. Am I supposed to use on of the above? I used `setupIT.bash` rather than `install.bash`.

Reply:
Right - these are alternatives (integration test installation vs normal installation)